qark/plugins/Manifest_File_Checks.py

```diff
@@ -58,66 +104,51 @@ def target(self, queue):
                 launch_mode_list.append(line)
 
             if any(re.findall(self.TASK_REPARENTING, line)):
-                PluginUtil.reportInfo(fileName, self.TaskReparentingIssue(fileName), res)
+                PluginUtil.reportInfo(file_name, task_reparenting(file_name), res)
+
+            if re.match(self.API_KEY_REGEX, line):
+                # Check if special character is present in the line. If "Yes, then ignore.
+                if re.match(self.SPECIAL_CHAR_REGEX, line):
+                    pass
+                else:
+                    if line not in api_key_list:
+                        api_key_list.append(line)
+
+            # Additional check for hardcoded api keys which matches the syntax most commonly used with google API_KEY
+            if any(re.findall(self.HARDCODED_API_KEY, line)):
+                if line not in api_key_list:
+                    api_key_list.append(line)
 
         # Arrange identified path variable and launch mode usage in column format
         path_variable = " \n".join(path_variable_list)
         launch_mode_variable = "\n".join(launch_mode_list)
+        api_key_variable = "\n".join(api_key_list)
 
         if path_variable_list:
-            PluginUtil.reportWarning(fileName, self.PathUsageIssue(path_variable), res)
+            PluginUtil.reportWarning(file_name, path_usage(path_variable), res)
 
         if launch_mode_list:
-            PluginUtil.reportInfo(fileName, self.LaunchModeIssue(launch_mode_variable), res)
+            PluginUtil.reportInfo(file_name, task_launch_mode(launch_mode_variable), res)
+
+        if api_key_list:
+            PluginUtil.reportInfo(file_name, hardcoded_api_key(api_key_variable), res)
 
-        # Check for google safebrowsing API
-        if "WebView" in f.splitlines():
-            if "EnableSafeBrowsing" and "true" not in f.splitlines():
-                PluginUtil.reportInfo(fileName, self.SafebrowsingIssueDetails(fileName), res)
+        # Check for google safe browsing API
+        if "WebView" in raw_file.splitlines():
+            if "EnableSafeBrowsing" and "true" not in raw_file.splitlines():
```

Comment:
This is not a thing in Python though I can understand what you are trying to do. Learning time!

```python
if "EnableSafeBrowsing" and "true" not in raw_file.splitlines():
```
Let's break this if statement into its logical pieces.
```python
if ("EnableSafeBrowsing") and ("true" not in raw_file.splitlines()):
```
The left side of the `if` statement is actually checking `bool("EnableSafeBrowsing")`, which will return `True` since strings in python are considered `True` if `string != ''` (i.e. not an empty string). The right side is actually doing what you want to do.

Change to 
```python
if "EnableSafeBrowsing" not in raw_file.splitlines() and "true" not in raw_file.splitlines():
```

And make sure to use the variable for `raw_file.splitlines()` so you don't have to recompute the splitlines everytime.